src/Board.py

```diff
@@ -45,10 +49,46 @@ def displayBoard(board):
         line = chr(x + UNICODE_FOR_A_CHAR)
 
         for i in range(len(board)):
-            
-            if board[x][i]:
+            if board[x][i] == SQUARE_SHIP:
+                line += " S "
+            elif board[x][i] == SQUARE_FAILED_SHOT:
+                line += " O "
+            elif board[x][i] == SQUARE_SUCCESS_SHOT:
                 line += " X "
             else:
                 line += "   "
 
         print(line)
+
+def shoot(coord, boards, currentPlayer):
+    boardsCopy = deepcopy(boards)
+    hit = 0
+
+    if (currentPlayer == 1):
```

Comment:
You can use a constant instead of a number (because I don't know if 1 means PLAYER_1 or PLAYER_2).

`if (currentPlayer == PLAYER_1)`

Reply:
I agree